src/ClassParser/ShiftClassBuilder.extension.st

```diff
@@ -0,0 +1,35 @@
+Extension { #name : #ShiftClassBuilder }
+
+{ #category : #'*ClassParser' }
+ShiftClassBuilder >> buildFromAST: aCDClassDefinitionNode [ 
+ 
+|package|
```

Comment:
Could you format this method ? It really does not follow any convention. See below